omega/omega-transport/omega-transport-servicecomb/src/main/java/org/apache/servicecomb/saga/omega/transport/servicecomb/SagaConsumerHandler.java

```diff
@@ -30,24 +30,23 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.annotations.VisibleForTesting;
-
 public class SagaConsumerHandler implements Handler {
 
   private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  private OmegaContext omegaContext;
+  private final OmegaContext omegaContext;
 
   public SagaConsumerHandler() {
+    OmegaContext context = null;
     try {
-      omegaContext = BeanUtils.getBean("omegaContext");
+      context = BeanUtils.getBean("omegaContext");
     } catch (NullPointerException npe) {
-      LOG.warn("The OmegaContext is not injected, The SagaConsumerHandler is disabled.");
+      LOG.warn("SagaConsumerHandler is not disabled, it's just cannot inject transaction ID.");
```

Comment:
The waring message should be "The OmegaContext is null, SagaConsumer cannot inject transaction ID"